setup.py

```diff
@@ -47,6 +47,14 @@
 __desc__ = 'Test Management Tool'
 __scripts__ = ['bin/tmt']
 
+# TODO: find out how to amend pyright, pre-commit and package requirements.
```

Comment:
Any chance setup.py data will eventually also be moved to pyproject.toml? This might be a good TODO to be part of that effort.  
If I missed any discussion about it, I'm sorry.  
I never tried to build RPMs from PEP 621-only project. Is that a blocker perhaps?

Reply:
Yeah, I would say 1. moving requirements into `pyproject.toml` is a worthy goal, and 2. RPM building is the great unknown, at least for me. I would love to see `setup.py` gone, `pyproject.toml` in action with Poetry, for example, but I have zero experience with RPM & `pyproject.toml` interaction, so this waits for someone to pick up.

I'm not sure whether such a move was discussed, I'm not with the project from the beginning and I did not propose it, given my inexperience, but it's a good idea to add to our weekly hacking sessions and find out. +1 from me in general, but it may show the RPM packaging would need way too much work/be impractical/be completely impossible.